hasura/graphql-engine
# Fix catalog version for v1.1.1 (close #4354)

fix #4354 
closes #4352 
### Description


Wrong catalog version (34) was provided for v1.1.1 in catalog_versions.txt. This PR fixes it to the correct version (31).
### Changelog


### Affected components



### Related Issues




### Solution and Design


Fix the catalog version for v1.1.1 to 31 in catalog_versions.txt

Modified server-upgrade tests to include downgrade tests as well.

### Steps to test and verify


1. Start graphql-engine with the current build
2. Stop and run downgrade command to v1.1.1
3. Start graphql-engine with version v1.1.1. The server should start without errors

### Limitations, known bugs & workarounds



### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
